app/assets/stylesheets/search.scss

```diff
@@ -13,16 +13,10 @@
 }
 
 .header {
-
-    .search-container {
-        margin-bottom: 5px;
```

Comment:
Just wanted to clarify: The reason this deletes a rule, is that there is a more general rule that handles this already.